autoscale/src/main/java/com/sequenceiq/periscope/service/EntitlementValidationService.java

```diff
@@ -33,7 +33,7 @@ public boolean autoscalingEntitlementEnabled(String accountId, String cloudPlatf
         return entitled;
     }
 
-    @Cacheable(cacheNames = "accountEntitlementCache", key = "{#accountId,#cloudPlatform}")
+    @Cacheable(cacheNames = "accountEntitlementCacheStopStart", key = "{#accountId,#cloudPlatform}")
```

Comment:
Was this somehow mixing up the 2 entitlements earlier? (one for whether autoscale is enabled or not, and the other for whether stop-start is enabled or not?)

Reply:
If so, good thing the CLI has the extra checks.

Reply:
Yes, I think previously this was using the same cache to determine if both the autoscaling entitlement as well as the stop-start scaling entitlement was enabled. Encountered this when writing endpointTests.